src/main/java/de/fuberlin/chaostesting/model/Response.java

```diff
@@ -22,14 +22,17 @@
 	private Date timestamp;
 	@Column(name="test_id")
 	private int test_id;
+	@Column(name="valid")
+	private String valid;
```

Comment:
This can be more accurately modeled by using a `boolean`. The string to be output should be coded in the jsp, and not in the database.